test/unit/actions-test.js

```diff
@@ -1,4 +1,6 @@
+import 'babel-polyfill';
```

Comment:
This seems like something we should import at the entry points to the application, rather than each file that needs it? Or perhaps I'm misunderstanding the polyfill use-case - I haven't used it much.